LICENSE-binary

```diff
@@ -0,0 +1,531 @@
+org.apache.kerby:kerb-core:1.0.1
+org.apache.kerby:kerb-crypto:1.0.1
+org.apache.kerby:kerb-identity:1.0.1
+org.apache.kerby:kerb-server:1.0.1
+org.apache.kerby:kerb-simplekdc:1.0.1
+org.apache.kerby:kerb-util:1.0.1
+org.apache.kerby:kerby-asn1:1.0.1
+org.apache.kerby:kerby-config:1.0.1
+org.apache.kerby:kerby-pkix:1.0.1
+org.apache.kerby:kerby-util:1.0.1
+org.apache.kerby:kerby-xdr:1.0.1
+org.apache.kerby:token-provider:1.0.1
+org.apache.yetus:audience-annotations:0.5.0
+org.apache.zookeeper:zookeeper:3.4.13
+org.codehaus.jackson:jackson-core-asl:1.9.13
+org.codehaus.jackson:jackson-jaxrs:1.9.13
+org.codehaus.jackson:jackson-mapper-asl:1.9.13
+org.codehaus.jackson:jackson-xc:1.9.13
+org.codehaus.jettison:jettison:1.1
+org.eclipse.jetty:jetty-annotations:9.3.27.v20190418
+org.eclipse.jetty:jetty-http:9.3.27.v20190418
+org.eclipse.jetty:jetty-io:9.3.27.v20190418
+org.eclipse.jetty:jetty-jndi:9.3.27.v20190418
+org.eclipse.jetty:jetty-plus:9.3.27.v20190418
+org.eclipse.jetty:jetty-security:9.3.27.v20190418
+org.eclipse.jetty:jetty-server:9.3.27.v20190418
+org.eclipse.jetty:jetty-servlet:9.3.27.v20190418
+org.eclipse.jetty:jetty-util:9.3.27.v20190418
+org.eclipse.jetty:jetty-util-ajax:9.3.27.v20190418
+org.eclipse.jetty:jetty-webapp:9.3.27.v20190418
+org.eclipse.jetty:jetty-xml:9.3.27.v20190418
+org.eclipse.jetty.websocket:javax-websocket-client-impl:9.3.27.v20190418
+org.eclipse.jetty.websocket:javax-websocket-server-impl:9.3.27.v20190418
+org.ehcache:ehcache:3.3.1
+org.objenesis:objenesis:1.0:compile
+org.xerial.snappy:snappy-java:1.0.5
+org.yaml:snakeyaml:1.16:
+org.wildfly.openssl:wildfly-openssl:1.0.7.Final
+
+
+--------------------------------------------------------------------------------
+This product bundles various third-party components under other open source
+licenses. This section summarizes those components and their licenses.
+See licenses-binary/ for text of these licenses.
+
+
+BSD 2-Clause
+------------
+
+hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/compress/lz4/{lz4.h,lz4.c,lz4hc.h,lz4hc.c}
+hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/fuse-dfs/util/tree.h
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/native/container-executor/impl/compat/{fstatat|openat|unlinkat}.h
+
+dnsjava:dnsjava:2.1.7
+org.codehaus.woodstox:stax2-api:3.1.4
+
+
+BSD 3-Clause
+------------
+
+hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/bloom/*
+hadoop-common-project/hadoop-common/src/main/native/gtest/gtest-all.cc
+hadoop-common-project/hadoop-common/src/main/native/gtest/include/gtest/gtest.h
+hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/util/bulk_crc32_x86.c
+hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/fuse-dfs/util/tree.h
+hadoop-tools/hadoop-sls/src/main/html/js/thirdparty/d3.v3.js
+hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/d3-3.5.17.min.js
+leveldb
```

Comment:
no version specified.

Reply:
I don't know the version of the leveldb libraries included in leveldbjni-all.jar.
There is no information in the GitHub repository: https://github.com/fusesource/leveldbjni
It seems that the leveldb version depends on the release manager's environment.

Reply:
Hadoop bundles [leveldbjni-all-1.8.jar](https://mvnrepository.com/artifact/org.fusesource.leveldbjni/leveldbjni-all/1.8) which contains pre-built libraries for each platform. [leveldb is build from the code in github.com/chirino/leveldb](https://github.com/fusesource/leveldbjni/blob/leveldbjni-1.8/readme.md#build-procedure) and its version is [1.13](https://github.com/chirino/leveldb/blob/leveldbjni-1.8/Makefile#L73-L74).

Reply:
Thank you for the information! Included the version.